redbot/core/commands/help.py

```diff
@@ -707,13 +708,10 @@ async def _delete_delay_help(
 
                 asyncio.create_task(_delete_delay_help(destination, messages, delete_delay))
         else:
-            # Specifically ensuring the menu's message is sent prior to returning
-            m = await (ctx.send(embed=pages[0]) if embed else ctx.send(pages[0]))
-            c = menus.DEFAULT_CONTROLS if len(pages) > 1 else {"\N{CROSS MARK}": menus.close_menu}
-            # Allow other things to happen during menu timeout/interaction.
-            asyncio.create_task(menus.menu(ctx, pages, c, message=m))
-            # menu needs reactions added manually since we fed it a messsage
-            menus.start_adding_reactions(m, c.keys())
+
+            await SimpleHybridMenu(
```

Comment:
This significantly changes the help menu behavior. I don't think this is a good thing. The 3 minute timeout also probably necessitates a max concurrency decorator on the help command and then also ensuring that separate invocations from context are also limited, which adds a lot more to what 3rd party replacements would also have to be aware of. I'd prefer this particular menu use not be included.